lib/functions/fs.js

```diff
@@ -43,40 +43,41 @@ module.exports = function(eyeglass, sass) {
   }
 
   function globFiles(directory, globPattern, includeFiles, includeDirectories, done) {
-      if (inSandbox(directory)) {
-        var globOpts = {
-          root: directory,
-          cwd: directory,
-          mark: true
-        };
-        glob(globPattern, globOpts, function(error, files) {
-          if (error) {
-            done(sass.types.Error(error.message));
-          } else {
-            var filesToReturn = [];
-            for (var i = 0; i < files.length; i++) {
-              var endsWithSlash = /\/$/.test(files[i]);
-              if (endsWithSlash && includeDirectories) {
-                if (!inSandbox(path.join(directory, files[i]))) {
-                  done(accessViolation(files[i]));
-                  return;
-                }
-                filesToReturn[filesToReturn.length] = files[i].slice(-files[i].length, -1);
-              }
-              if (!endsWithSlash && includeFiles) {
-                if (!inSandbox(path.join(directory, files[i]))) {
-                  done(accessViolation(files[i]));
-                  return;
-                }
-                filesToReturn[filesToReturn.length] = files[i];
-              }
+    if (inSandbox(directory)) {
+      var globOpts = {
+        root: directory,
+        cwd: directory,
+        mark: true
+      };
+      glob(globPattern, globOpts, function(error, files) {
+        if (error) {
+          done(sass.types.Error(error.message));
+          return;
```

Comment:
The only change here is to `return` in this conditional, which allows us to remove the `else` clause, reducing the level of nesting and avoiding the `max-depth` warning.

The rest of the change shown here was just correcting the indent.